README.md

```diff
@@ -1,7 +1,9 @@
 # CakePHP Localized plugin
-[![Build Status](https://travis-ci.org/cakephp/localized.svg?branch=master)](http://travis-ci.org/cakephp/localized)
-[![License](https://poser.pugx.org/cakephp/localized/license.svg)](https://packagist.org/packages/cakephp/localized)
-[![Total Downloads](https://poser.pugx.org/cakephp/localized/d/total.svg)](https://packagist.org/packages/cakephp/localized)
+
+[![Build Status](https://img.shields.io/travis/com/cakephp/localized/master?style=flat-square)](https://travis-ci.com/cakephp/localized)
```

Comment:
Any reason you dont want to use the default travis ones?

Reply:
Just consistency. Must of our repos used this format and I copied from them. Easier for me to copy around.